src/view/channel.js

```diff
@@ -171,6 +174,13 @@ const ChannelSummary = ({
         {channelBalanceOpenLabel} {unitLabel}
       </Text>
     </View>
+    <View style={summaryStyles.box}>
+      <Alert type="inactive" style={summaryStyles.alert} />
+      <Text style={summaryStyles.txt}>Inactive</Text>
+      <Text style={[summaryStyles.txt, summaryStyles.total]}>
+        {channelBalanceInactiveLabel} {unitLabel}
+      </Text>
+    </View>
```

Comment:
I wouldn't add another summary here. Inactive balance should be included in the open balance.

Reply:
fixed :)

Reply:
Really sorry. I confused this. Just retested and it's better with the separate summary. I reverted the commit.